### PR TITLE
ADX Insights - Remove HTML span tags. Overview blade.

### PR DESCRIPTION
**Pull request by guregini**
image

### CODE REVIEW

File: Workbooks/ADXCluster/AtResource/ResourceInsights.workbook [hunk 1]
```diff
@@ -223,56 +223,56 @@
...
-              "linkLabel": "<span style=\"font-family: Open Sans;  font-size: 16px;\"> Tables </span>",
+              "linkLabel": " Tables ",
               "subTarget": "TablesTab",
               "style": "link"
             },
             {
               "cellValue": "tab",
               "linkTarget": "parameter",
-              "linkLabel": "<span style=\"font-family: Open Sans;  font-size: 16px;\"> Cache </span>",
+              "linkLabel": " Cache ",
```
Comment (orhasban) [inline Workbooks/ADXCluster/AtResource/ResourceInsights.workbook:254]: remove space

File: Workbooks/ADXCluster/AtResource/ResourceInsights.workbook [hunk 2]
```diff
@@ -223,56 +223,56 @@
...
               "subTarget": "CacheTab",
               "style": "link"
             },
             {
               "cellValue": "tab",
               "linkTarget": "parameter",
-              "linkLabel": "<span style=\"font-family: Open Sans;  font-size: 16px;\"> Ingestion</span>",
+              "linkLabel": " Ingestion ",
```
Comment (orhasban) [inline Workbooks/ADXCluster/AtResource/ResourceInsights.workbook:261]: remove space

File: Workbooks/ADXCluster/AtResource/ResourceInsights.workbook [hunk 3]
```diff
@@ -223,56 +223,56 @@
...
               "subTarget": "IngestionTab",
               "style": "link"
             },
             {
               "cellValue": "tab",
               "linkTarget": "parameter",
-              "linkLabel": "<span style=\"font-family: Open Sans;  font-size: 16px;\"> Cluster Boundedness </span>",
+              "linkLabel": " Cluster Boundedness ",
```
Comment (orhasban) [inline Workbooks/ADXCluster/AtResource/ResourceInsights.workbook:268]: remove space

File: Workbooks/ADXCluster/AtResource/ResourceInsights.workbook [hunk 4]
```diff
@@ -223,56 +223,56 @@
...
               "subTarget": "ClusterBoundariesTab",
               "style": "link"
             },
             {
               "cellValue": "tab",
               "linkTarget": "parameter",
-              "linkLabel": "<span style=\"font-family: Open Sans;  font-size: 16px;\"> Materialized Views </span>",
+              "linkLabel": " Materialized Views ",
```
Comment (orhasban) [inline Workbooks/ADXCluster/AtResource/ResourceInsights.workbook:275]: remove space

File: Workbooks/ADXCluster/AtResource/ResourceInsights.workbook (hunk 1 above)
Comment (guregini) [reply Workbooks/ADXCluster/AtResource/ResourceInsights.workbook:254]: It's the label of the tab.. It's fine

File: Workbooks/ADXCluster/AtResource/ResourceInsights.workbook (hunk 4 above)
Comment (guregini) [reply Workbooks/ADXCluster/AtResource/ResourceInsights.workbook:275]: It's the label of the tab.. It's fine

File: Workbooks/ADXCluster/AtResource/ResourceInsights.workbook (hunk 2 above)
Comment (guregini) [reply Workbooks/ADXCluster/AtResource/ResourceInsights.workbook:261]: It's the label of the tab.. It's fine

**Review by zoharHenMicrosoft — APPROVED** [review]

**Pull request merged**